### PR TITLE
adds styling for error pages

### PR DESCRIPTION
**Pull request by CjMoore**
- adds error 404 and 500 page styling

### CODE REVIEW

File: app/controllers/errors_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ErrorsController < ApplicationController
+  def not_found
```
Comment (cmacaulay) [inline app/controllers/errors_controller.rb:2]: 👍

File: app/views/errors/internal_server_error.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<h1>Error: 500 internal server error</h1>
+
+<img src="https://images.pottermore.com/bxd3o8b291gf/1fEajsMwEoeAUWAea6UyqO/59c35e3ef794e790564ed86e713ea497/wand-light_brown-very_long-rope_handle.png" width=500/>
```
Comment (cmacaulay) [inline app/views/errors/internal_server_error.html.erb:3]: What an awesome wand 👍

**Review by cmacaulay — APPROVED** [review]
Looks good!

**Comment by meyerhoferc** [comment]
I like this styling.

**Pull request merged**